### PR TITLE
 Include question to answer

### PR DESCRIPTION
**Pull request by hmtuan196**
Add space if i is divisible by 15 in question 16

### CODE REVIEW

**Comment by hmtuan196** [comment]
Also correct the link of question 16

File: README.md [hunk 1]
```diff
@@ -207,7 +223,7 @@ const helloBitrep = () => {
       output += "hello";
     }
     if (!(i % 5)) {
-      output += "bitrep";
+      output = output === "" ? "bitrep" : `${output} bitrep`;
```
Comment (shirai91) [inline README.md]: This check is ugly, i'm thinking about just append string with space ahead
```javascript
output += " bitrep";
```
and print trimmed string with
```javascript
 console.log(output.trim());
``` 
what do you think?

Comment (hmtuan196) [reply README.md]: If i is divisible by 5 but not divisible by 3, then the `output` always has space at the beginning. Is it right?

Comment (hmtuan196) [reply README.md]: ah yes, you right :))

**Pull request merged**